cli/command/registry.go

```diff
@@ -69,9 +78,17 @@ func GetDefaultAuthConfig(cfg *configfile.ConfigFile, checkCredStore bool, serve
 			}, err
 		}
 	}
-	authCfg.ServerAddress = serverAddress
-	authCfg.IdentityToken = ""
-	return registrytypes.AuthConfig(authCfg), nil
+
+	return registrytypes.AuthConfig{
+		Username:      authCfg.Username,
+		Password:      authCfg.Password,
+		ServerAddress: serverAddress,
+
+		// TODO(thaJeztah): Are these expected to be included?
+		Auth:          authCfg.Auth,
```

Comment:
More so, because the `Auth` field .. I _think_ is the base64 encoded representation of `Username:Password`, so either the structured fields OR only the `Username:Password` should likely be sent.